static/docs/commands-reference/version.md

```diff
@@ -14,14 +14,14 @@ usage: dvc version [-h] [-q | -v]
-| ------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [`DVC version`](#components-of-dvc-version) | Version of DVC (along with a Git commit hash in case of a development version)                                                                            |
-| `Python version`                            | Version of the Python being used for the project in which DVC is initialized                                                                              |
-| `Platform`                                  | Information about the operating system of the machine                                                                                                     |
-| [`Binary`](#output-of-binary)               | Shows whether the package is installed from a binary release or source                                                                                    |
-| `Cache`                                     | [Type of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported between the DVC workspace and the cache directory |
-| `Filesystem type`                           | Shows the filesystem type (eg. ext4, FAT, etc.) and mount point of workspace and the cache directory                                                      |
+| Type                                        | Detail                                                                                                                                                                 |
+| ------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`DVC version`](#components-of-dvc-version) | Version of DVC (along with a Git commit hash in case of a development version)                                                                                         |
+| `Python version`                            | Version of the Python being used for the project in which DVC is initialized                                                                                           |
+| `Platform`                                  | Information about the operating system of the machine                                                                                                                  |
+| [`Binary`](#output-of-binary)               | Shows whether the package is installed from a binary release or source                                                                                                 |
+| `Cache`                                     | [Type of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported between the DVC workspace and the <abbr>cache</abbr> directory |
```

Comment:
I'm not sure I like the idea of using <abbr> </abbr> for each term we need to highlight, but have no better ideas from the top of my head ... @jorgeorpinel anything comes to your mind?

Reply:
@shcheklein I don't exactly get what you are saying. You don't want to use `abbr` tags and let the code automatically detect the keywords?

Reply:
It would be ideal if tooltips could be added automatically by the docs engine. Being careful not to interfere with any existing automatic parsing like the `dvc cmd ref` links.

Reply:
@jorgeorpinel Okay, I will try to figure out.

Reply:
@jorgeorpinel I think it will be better if we manually indicate which terms we need to highlight using `abbr` tags since automatic detection will require checking each word in a markdown file and checking if it exists in `glossary.json`.

Reply:
I agree this is a complicated feature, lets discuss with Ivan (:

Reply:
Yeah, don't see a way to make it automatic. Let's just keep on a writer discretion to put the <abbr> tag when it makes sense. Like with the DVC-file link we discussed with @jorgeorpinel it probably does not make sense to put it all around the document. Just a few places? 

We can also show the terms on the left side. And show tooltip when you hover.